ckanext/canada/commands.py

```diff
@@ -61,6 +61,7 @@ class CanadaCommand(CkanCommand):
         paster canada portal-update <portal.ini>
```

Comment:
```suggestion
        paster canada portal-update <portal.ini> -u <user>
```
I would put it here instead and not use `[` `]` because it's not an optional parameter